tests/playwright/sanity/includes/widgets/image.test.js

```diff
@@ -0,0 +1,32 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../../../pages/wp-admin-page.js' );
+
+test( 'Lightbox image captions aligned center', async ( { page }, testInfo ) => {
+	// Arrange.
+	const wpAdmin = new WpAdminPage( page, testInfo );
+	const editor = await wpAdmin.useElementorCleanPost();
+
+	// Act.
+	const frame = page.frameLocator( '#elementor-preview-iframe' );
+	await editor.addWidget( 'image' );
+
+	await page.locator( '.elementor-control-media__preview' ).click();
+	await page.getByRole( 'tab', { name: 'Media Library' } ).click();
+	await page.setInputFiles( 'input[type="file"]', './tests/playwright/resources/elementor1.png' );
+	await page.locator( '#attachment-details-title' ).fill( 'Elementor Logo (title)' );
```

Comment:
Do we need to add alt text and caption?
I don't think that we are asserting these.

Reply:
@hein-obox  By default Lightbox has a Title and Description but caption and Alt tags can be assigned to either of these in SiteSettings => Lightbox. I've tested Alt, Caption by hand. I'll remove them from the test.